doc/source/index.rst

```diff
@@ -0,0 +1,21 @@
+.. pandas-mentoring documentation master file, created by
+   sphinx-quickstart on Tue Jul 30 15:08:15 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
```

Comment:
You don't necessary need to leave this comment. Not a problem at all to leave it you find it useful or you prefer, up to you.

Reply:
Will remove